quickwit/quickwit-index-management/src/garbage_collection.rs

```diff
@@ -44,6 +45,12 @@ use tracing::{error, instrument};
 /// The maximum number of splits that the GC should delete per attempt.
 const DELETE_SPLITS_BATCH_SIZE: usize = 10_000;
 
+pub struct GcMetrics {
+    pub deleted_splits: IntCounter,
+    pub deleted_bytes: IntCounter,
+    pub failed_splits: IntCounter,
```

Comment:
```suggestion
    deleted_splits: IntCounter,
    deleted_bytes: IntCounter,
    failed_splits: IntCounter,
```

if you do the refactoring suggested below.

Reply:
i don't understand this suggestion. GcMetrics is provided from another crate, we need these fields public to create the struct

Reply:
ok